res/controllers/Numark-Mixtrack-Platinum-scripts.js

```diff
@@ -1413,13 +1413,13 @@ MixtrackPlatinum.vuCallback = function(value, group, control) {
             midi.sendShortMsg(0xBF, 0x45, level);
         }
     }
-    else if (group == '[Master]' && control == 'VuMeterL') {
+    else if (group == '[Main]' && control == 'vu_meter_left') {
```

Comment:
yeah, please fix the indent style while ur touching this line

Reply:
Actually, it is not the indent what it is complaining about.
It is complaining that the Script uses single quotes ' for strings rather than double quotes " .